Makefile

```diff
@@ -308,15 +308,16 @@ DATAPLANE_KUTTL_NAMESPACE ?= dataplane-kuttl-tests
 DATAPLANE_DEFAULT_GW      ?= 192.168.122.1
 
 # Manila
-MANILA_IMG            ?= quay.io/openstack-k8s-operators/manila-operator-index:latest
-MANILA_REPO           ?= https://github.com/openstack-k8s-operators/manila-operator.git
-MANILA_BRANCH         ?= main
-MANILA                ?= config/samples/manila_v1beta1_manila.yaml
-MANILA_CR             ?= ${OPERATOR_BASE_DIR}/manila-operator/${MANILA}
+MANILA_IMG              ?= quay.io/openstack-k8s-operators/manila-operator-index:latest
+MANILA_REPO             ?= https://github.com/openstack-k8s-operators/manila-operator.git
+MANILA_BRANCH           ?= main
+MANILA                  ?= config/samples/manila_v1beta1_manila.yaml
+MANILA_CR               ?= ${OPERATOR_BASE_DIR}/manila-operator/${MANILA}
 # TODO: Image customizations for all Manila services
-MANILA_KUTTL_CONF     ?= ${OPERATOR_BASE_DIR}/manila-operator/kuttl-test.yaml
-MANILA_KUTTL_DIR      ?= ${OPERATOR_BASE_DIR}/manila-operator/tests/kuttl/tests
-MANILA_KUTTL_TIMEOUT  ?= 180
+MANILA_KUTTL_CONF       ?= ${OPERATOR_BASE_DIR}/manila-operator/kuttl-test.yaml
+MANILA_KUTTL_DIR        ?= ${OPERATOR_BASE_DIR}/manila-operator/tests/kuttl/tests
+MANILA_KUTTL_TIMEOUT    ?= 180
```

Comment:
Is this variable used anywhere?

Reply:
Occasionally we use it for triage, currently not used in the file. Can be removed until needed.

Reply:
Perhaps we can remove it for now, otherwise it might give the user the impression that it actually currently does something.

Reply:
No problem, done.